src/Bridge/Symfony/Bundle/Resources/config/elasticsearch.xml

```diff
@@ -45,7 +45,6 @@
         <service id="ApiPlatform\Core\Bridge\Elasticsearch\Api\IdentifierExtractorInterface" alias="api_platform.elasticsearch.identifier_extractor" />
 
         <service id="api_platform.elasticsearch.name_converter.inner_fields" class="ApiPlatform\Core\Bridge\Elasticsearch\Serializer\NameConverter\InnerFieldsNameConverter" public="false">
-            <argument type="service" id="api_platform.name_converter" on-invalid="ignore" />
```

Comment:
@meyerbaptiste if I don't do this the default name converter doesn't convert query filters from Camel to snake and it fails. Do you have another solution in mind?

Reply:
You set the default "fallbackNameConverter" of the MetadataNameConverter to be "api_platform.name_converter" by default.

If people choose CamelCaseToSnakeCase then it will work fine.

Reply:
Yes it's now working properly as stated in #2677:

If MetadataAwareNameConverter it's used
If `api_platform.name_converter` exists it's used as fallback
If symfony's `name_converter` is configured it'll be used as fallback
If none is given, the `MetadataAwareNameConverter` will not do any conversion